modules/ROOT/partials/misc/admon-requires-7.1v.adoc

```diff
@@ -0,0 +1 @@
+NOTE: This feature is only available for {productname} 7.1.0 and later.
```

Comment:
Looking at other required version admonitions, typically we don't include the patch version (the `.0`) unless the new feature was released in a patch version.